jveillet/tentacles
# [ISSUE-44] Issue with repositories selection

## Description
Changed the general behavior of the "select all repositories" checkbox. If we select individually all the repositories checkbxes one by one until reaching every one of them, it should check the "select all repositories" checkbox too.

## Related Issue
https://github.com/jveillet/tentacles/issues/44

## Motivation and Context
Checking the "select all repositories" checkbox wasn't checking/unchekcing all the repositories checkboxes. Plus, checking/uncheking of all the repositories checkoxes didn't affect the "select all" checkbox.

## How Has This Been Tested?
On the repositories page:
- Check/uncheck the "select all" checkbox to see the repositories be checked/unchecked.
- Check individually all the repositories checkboxes and see the "select all" checkbox update accordingly.
- Unselect only one or two repositories to see the "select all" checkbox update accordingly.
- Select again the "select all" checkox and see the missing repositories checkboxes be checked again.

## Screenshots (if appropriate):

## Types of changes
  change)

## Checklist:
